src/MissingPrisoners/MissingPrisonersSearch.js

```diff
@@ -56,12 +61,17 @@ const MissingPrisonersSearch = ({
             <option key="MORNING" value="AM">
```

Comment:
don't you need `!isTodayOrAfter` for the `am` option too, otherwise it will be shown for tomorrow won't it?

Reply:
We don't cater for future dates on this.  The date picker shows today and past dates.